raiden/network/rpc/client.py

```diff
@@ -198,6 +198,16 @@ def decode_topic(topic):
     return int(topic[2:], 16)
 
 
+def estimate_and_transact(self, callobj, *args):
```

Comment:
`self` is a convention to be used for methods of  python classes to point to the instantiated object of the class. You should never used it as a simple identifier outisde of a class like that.

I had assumed all calls you were making were from the same class which is why in my snippet I used `self`.

Looking at the required places where this function needs to be used that is not the case (it is used in many different classes) and as such I would suggest to change the name of the variable to avoid confusions.

Change `self` to `classobject`.